src/reusecore/Search/index.js

```diff
@@ -0,0 +1,23 @@
+import { FaSearch } from "@react-icons/all-files/fa/FaSearch";
+import React from "react";
+import { Search } from "./searchbox.style";
```

Comment:
Better name it `SearchWrapper` to align with the codebase naming structure.

Reply:
This is understandably an issue I am having @subhradwip01 too. I don't know why this is happening right now giving the fact the graphQL  query is working as it should. I will continue looking into it.

Reply:
Everything now works with the latest commit.